README.md

```diff
@@ -219,3 +219,53 @@ MaxText supports training and inference of various open models. Follow user guid
 
 * [Gemma](https://ai.google.dev/gemma): a family of open-weights Large Language Model (LLM) by [Google DeepMind](https://deepmind.google/), based on Gemini research and technology. \
 You can run decode and finetuning using instructions mentioned [here](https://github.com/google/maxtext/blob/main/getting_started/Run_Gemma.md).
+
+## Automatically Upload Logs to Vertex Tensorboard
```

Comment:
Let's move this in a standalone file. I'm trying to simplify the README. We will link to the standalone file in the primary README.

https://github.com/google/maxtext/tree/main/getting_started -- place for various explainers including this one.

Reply:
Done!